maintainers-circle/README.md

```diff
@@ -51,7 +51,6 @@ In-person as soon as we can at the next face-to-face KubeCon (TBD)
 
 ## When  
 Every other Contributor Strategy meeting:*  
```

Comment:
Did you intend to change the maintainers circle readme? It looks like an accident that got included with the wrong PR?